src/adapters/postgres/fields-adapter.ts

```
@@ -2,304 +1,1773 @@
+    tableName,
+    whereClause?: any,
+    offset?: any,
+    limit?: any,
+    order?: any,
+    optionName?: any
+  ) {
+    const orderCond = order || "";
+    const offsetCond = offset ? `offset ${offset}` : "";
+    const limitCond = limit ? `limit ${limit}` : "";
+    let whereCond = `WHERE `;
+    whereCond = whereClause ? (whereCond += `${whereClause}`) : "";
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)